tools/eslint-rules/no-definegetter-definesetter.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * @fileoverview Rule to flag usage of __defineGetter__ and __defineSetter__
+ * @author Rich Trott
+ */
+
+'use strict';
+
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+
+module.exports = {
+  create: function(context) {
+    const disallowed = ['__defineGetter__', '__defineSetter__'];
+
+    return {
+      MemberExpression: function(node) {
+        var prop;
+        if (node.property) {
+          if (node.property.type === 'Identifier' && !node.computed) {
```

Comment:
`!node.computed`... so `obj['__defineGetter__'] = getter` is the escape hatch? =)